microsoft/testsuites/dpdk/dpdkutil.py

```diff
@@ -273,6 +279,7 @@ def initialize_node_resources(
     variables: Dict[str, Any],
     pmd: str,
     sample_apps: Union[List[str], None] = None,
+    enable_gibibyte_hugepages: bool = False,
```

Comment:
now, all distros won't enable_gibibyte_hugepages

Reply:
This is correct, I'm going to add specific tests for 1GB hugepages later.

Reply:
Added a second commit to the PR to add the 1GB hugepage tests so we don't lose coverage.

Reply:
please handle it in another PR